examples/flow.cpp

```diff
@@ -356,9 +356,9 @@ try
 
     // Solver for Newton iterations.
     std::unique_ptr<NewtonIterationBlackoilInterface> fis_solver;
-    if (param.getDefault("use_interleaved", false)) {
+    if (param.getDefault("use_interleaved", true)) {
         fis_solver.reset(new NewtonIterationBlackoilInterleaved(param, parallel_information));
-    } else if (param.getDefault("use_cpr", true)) {
+    } else if (param.getDefault("use_cpr", false)) {
```

Comment:
To clarify: I think the interleaved can be the default, but if we choose actively not to use it, the CPR should be default, meaning I want line 361 to have a "true" and the rest of the PR as it is.

Perhaps we should (after this) replace these bool flags by a string parameter (linear_solver_strategy perhaps?), which would be clearer.